py/server/tests/test_pyfunc_param_null.py

```diff
@@ -0,0 +1,170 @@
+                    self.assertNotIn("-inf", res3.to_string().lower())
+                else:
+                    self.assert_table_equals(res2, res3)
+
+                with self.assertRaises(AssertionError):
+                    self.assert_table_equals(res, res2)
+
+                exec(f"del {null_name}", globals())
+
+    def test_dh_null_to_nan_scalar(self):
+        col1_formula = "Col1 = i % 10"
+        for data_type, null_name in _J_TYPE_NULL_MAP.items():
+            col2_formula = f"Col2 = i % 3 == 0? {null_name} : ({data_type})i"
+            with self.subTest(data_type):
+                np_type = _J_TYPE_NP_DTYPE_MAP[data_type]
+                func = f"""
+@dh_null_to_nan
+def custom_func4(col: {np_type}) -> bool:
+    if np.isnan(col):
+        return True
+    else:
+        return False
+"""
+                exec(func, globals())
+                with self.subTest(data_type):
+                    tbl = empty_table(100).update([col1_formula, col2_formula])
+                    res = tbl.update("Col4 = custom_func4(Col2)")
+                    self.assertEqual(4, res.to_string().count("true"))
+
+                @dh_null_to_nan
+                @dh_vectorize
+                def custom_func5(col) -> bool:
+                    if np.isnan(col):
+                        return True
+                    else:
+                        return False
+
+                with self.subTest(data_type):
+                    tbl = empty_table(100).update([col1_formula, col2_formula])
+                    res = tbl.update("Col5 = custom_func5(Col2)")
+                    self.assertEqual(4, res.to_string().count("true"))
+
+    def test_dh_null_to_na_array(self):
+        @dh_null_to_na
+        def custom_func(c0l1, col2):
+            return col2.mean(skipna=False)
+
+        @dh_null_to_na
+        def custom_func2(col1, col2):
+            return col2.mean(skipna=True)
+
+        col1_formula = "Col1 = i % 10"
+        for data_type, null_name in _J_TYPE_NULL_MAP.items():
+            col2_formula = f"Col2 = i % 3 == 0? {null_name} : ({data_type})i"
+            with self.subTest(data_type):
+                tbl = empty_table(100).update([col1_formula, col2_formula]).group_by("Col1")
+
+                exec(f"from deephaven.constants import {null_name}", globals())
+                custom_func3_str = f"lambda col1, col2: float(np.mean([c for c in col2 if c != {null_name}]))"
+                custom_func3 = eval(custom_func3_str, globals(), locals())
+
+                res = tbl.update("Col3 = custom_func(Col1, Col2)")
+                res2 = tbl.update("Col3 = custom_func2(Col1, Col2)")
+                res3 = tbl.update("Col3 = custom_func3(Col1, Col2)")
+                if data_type in ["double", "float"]:
+                    if data_type == "double":
+                        self.assert_table_equals(res2, res3)
+                else:
+                    self.assert_table_equals(res2, res3)
```

Comment:
Isn't this basically the same as:
```
if data_type != "float":
    do test...
else:
    don't test...
````
If so, I think this presentation is more clear.  Also why not testing float?